front/app/services/initiativeFiles.ts

```diff
@@ -18,10 +18,6 @@ export interface IInitiativeFileData {
   };
```

Comment:
There is still a dependency on this file in the useRemoteFiles hook. We can delete this when that is done